rpcs3/Emu/games_config.cpp

```diff
@@ -64,6 +74,8 @@ bool games_config::add_game(const std::string& key, const std::string& path)
 
 bool games_config::save()
 {
+	std::lock_guard lock(m_mutex);
```

Comment:
save shouldn't occur cuncurrently, I siggest using trylock() here and if it fails return prematurely and log an error.

Reply:
There is a Szenario where the UI is refreshed while a game triggers a boot.
(Whether or not this happens on the main thread anyway is not relevant due to future proofing)

Reply:
Which is clearly invalid they cannot coexist.

Reply:
The decision to use AddGamesFromDir for /games was a mistake.

Reply:
It doesn't matter with the other PR

Reply:
It's a rare edge case that should never happen